pkg/uploader/provider/provider.go

```diff
@@ -42,3 +50,21 @@ type Provider interface {
 	// Close which will close related repository
 	Close(ctx context.Context)
 }
+
+func NewUploaderProvider(
+	ctx context.Context,
+	legacyUploader bool,
+	repoIdentifier string,
+	bsl *velerov1api.BackupStorageLocation,
+	credGetter *credentials.CredentialGetter,
+	repoKeySelector *v1.SecretKeySelector,
+	configFile string,
```

Comment:
This parameter is not used, remove it, please

Reply:
yeah, I've changed